src/src/rx_main.cpp

```diff
@@ -318,7 +334,16 @@ bool ICACHE_RAM_ATTR HandleFHSS()
     }
 
     alreadyFHSS = true;
-    Radio.SetFrequencyReg(FHSSgetNextFreq());
+
+    if (geminiMode)
+    {
+        Radio.SetFrequencyReg(FHSSgetNextFreq(), SX12XX_Radio_1);
+        Radio.SetFrequencyReg(FHSSgetGeminiFreq(), SX12XX_Radio_2);
```

Comment:
What do you think about alternating the antennas on RX, instead of fixed freq?

like
```
if (OtaNonce %2 == 0)
{
        Radio.SetFrequencyReg(FHSSgetNextFreq(), SX12XX_Radio_1);
        Radio.SetFrequencyReg(FHSSgetGeminiFreq(), SX12XX_Radio_2);
}
else
{
        Radio.SetFrequencyReg(FHSSgetGeminiFreq(), SX12XX_Radio_1);
        Radio.SetFrequencyReg(FHSSgetNextFreq(), SX12XX_Radio_2);
}
```

?

Reply:
For sure its something we should test in a future PR, along with switching between radios when in single antenna mode.